zfs.yaml

```diff
@@ -123,11 +123,11 @@ test:
           /usr/lib/libnvpair.so.3.0.0
           /usr/lib/libuutil.so.3
           /usr/lib/libuutil.so.3.0.0
-          /usr/lib/libzfs.so.4
-          /usr/lib/libzfs.so.4.1.0
+          /usr/lib/libzfs.so.6
```

Comment:
thank you for fixing this, I think we can consider updating it to use the following:
```yaml
    - uses: test/ldd-check
      with:
        packages: ${{package.name}}
```
this way, we won't have to do this manually going forward. 💭

Reply:
Thanks for the review! I've just pushed up the requested change.